### DISCUSSION TITLE
三月更新 - 3

### DISCUSSION THREAD

**Discussion by Richasy** [Richasy/Bili.Uwp#846] [通知]
## 更改记录

🎉 **这次更新支持8K和HDR视频流获取，感兴趣的小伙伴可以打开支持HDR的视频，并切换分辨率至 `HDR 真彩`。另外，如果你的设备不支持HDR，请勿尝试，可能会放不出来，或极其卡顿 ౿(།﹏།)૭** 🎉

## 🚀 功能

- 在视频播放结束时恢复标准视图 @Richasy (#845)
- 播放速率功能调整 @Richasy (#843)
    - 添加了一个实验性设置，在设置页的播放器控制设置里可以开启 `播放速率增强`，将播放速率上限提高至6倍速
- 支持优先AV1解码 @Richasy (#842) （需提前在微软应用商店下载AV1视频解码扩展）
- 支持视频截图 @Richasy (#821) 
    - 可以在设置页面里配置截图行为，快捷键是Shift+S，播放控制器中也添加了一个按钮

## 🐛 Bug 修复

- 修改错误的issue地址 @XiaoMouz (#833)
- 修复视频发布时间的时差问题 @Richasy (#844)
- 修改错误的仓库地址 @Misaka-L (#829)

<hr /><em>This discussion was created from the release <a href='https://github.com/Richasy/Bili.Uwp/releases/tag/v2.2203.3.0'>三月更新 - 3</a>.</em>

**Comment by jiyingxuan** [comment]
赞美开发者！

**Comment by lns103** [comment]
居然还支持了杜比视界，支持作者
![image](https://user-images.githubusercontent.com/49310852/159104780-3be4c5ae-dab7-48e5-b911-755f4cb4b245.png)

**Reply by midnightzbc** [reply]
App_2.2202.1.0_Test 之后的版本安装后就打不开了。。打开之后卡在进去主界面之前。。系统是win10 ltsc。不知道是啥情况?

**Reply by Richasy** [reply]
@midnightzbc 参考这个：#796

**Reply by midnightzbc** [reply]
@Richasy 谢谢~后续更新会解决这个问题吗？看了下网上评价2021ltsc还是有点毛病，不太想升。。

**Reply by Richasy** [reply]
@midnightzbc 目前还不确定具体是哪个原因导致的这个问题，但未来版本应该只会升不会降，如果有必要，未来可能会放弃对 Windows 10 的支持，以避免写一些版本适应性代码

**Reply by midnightzbc** [reply]
@Richasy 好的，非常感谢~
